src/app/src/components/FacilityListsTable.jsx

```diff
@@ -11,6 +11,21 @@ import TableCell from '@material-ui/core/TableCell';
 import { facilityListPropType } from '../util/propTypes';
 import { makeFacilityListItemsDetailLink } from '../util/util';
 
+import COLOURS from '../util/COLOURS';
+
+const facilityListsTableStyles = Object.freeze({
+    inactiveListStyles: Object.freeze({
+        cursor: 'pointer',
+    }),
+    activeListStyles: Object.freeze({
+        backgroundColor: 'rgb(222, 242, 254)',
```

Comment:
In the mockup this was `#f3fafe`, not `#def2fe`

Reply:
Cool, I'll change this before merging.